### PR TITLE
#282 General page on dast tools.

### PR DESCRIPTION
**Pull request by sydseter**
**Summary** :  

Fixes: #282

**Description for the changelog** :  

Changing the ZAP page into a page about DAST in general

**Other info** :  

More could be added, but this will give a good direction for now.

### CODE REVIEW

**Comment by thc202** [comment]
Not opposing the change but would be good this change was not made based on FUD.

**Comment by sydseter** [comment]
true, but I think that is clarified now in https://github.com/OWASP/www-project-developer-guide/issues/282 
Perhaps something feels amiss?

**Comment by thc202** [comment]
Not with this PR, just that the issue title and description is still wrong/false.

**Comment by jgadsden** [comment]
@thc202 the issue title is based on the new ZAP logo and the [Checkmarx press release](https://checkmarx.com/press-releases/checkmarx-joins-forces-with-zap-to-supercharge-dynamic-application-security-testing-dast-for-the-enterprise-and-enhance-community-growth/)

Checkmarx is very clear about what is still free to the ZAP community and what will now be available only to Checkmarx customers

Screenshot 2024-09-27 at 14 32 09

**Comment by jgadsden** [comment]
@sydseter you are probably already editing these, but these files also need changes: 

- draft/08-verification/02-tools/00-toc.md
- draft/08-verification/00-toc.md
- draft/02-toc.md
- _data/draft.yaml

**Comment by thc202** [comment]
https://github.com/OWASP/www-project-developer-guide/pull/283#issuecomment-2379300265

If it's very clear, you should have no problems answering what features are becoming paid-for in ZAP (as the issue still falsely states: `In addition some ZAP features will now become paid-for`).

File: draft/08-verification/02-tools/01-dast.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+DAST tools are programs which communicates with a web application through the web front-end in order to identify potential
+security vulnerabilities in the web application and architectural weaknesses. It performs a black-box test. Unlike static
+application security testing tools, DAST tools do not have access to the source code and therefore detect vulnerabilities
+by actually performing attacks.
+
+#### Different DAST tools
+
+The OWASP Community projects contains a [list of DAST tools][dast] can be used to conduct DAST. All of these tools have
+their own strengths and weaknesses. If you are interested in the effectiveness of DAST tools, check out the
+[OWASP Benchmark][benchmark] project, which is scientifically measuring the effectiveness of all types of vulnerability
```
Comment (kingthorin) [inline draft/08-verification/02-tools/01-dast.md:42]:
```suggestion
[OWASP Benchmark][benchmark] project, which attempts to scientifically measure the effectiveness of all types of vulnerability
```

Comment (sydseter) [reply draft/08-verification/02-tools/01-dast.md]: said and done.

**Comment by jgadsden** [comment]
@sydseter the final error in the link checker will be corrected when the draft is promoted to release, so do not worry about trying to fix it

**Review by jgadsden — APPROVED** [review]
thanks for doing this work (and accepting the challenge) @sydseter 
everything looks like it is in place, so please go ahead and merge

**Pull request merged**